src/Helpers/FileHelper.cs

```diff
@@ -27,8 +28,22 @@ public class FileHelper
     {
         public static IEnumerable<string> GetImplementationFiles()
         {
-            var classFiles = Directory.EnumerateFiles(Utils.GaugeProjectRoot, "*.cs",
-                SearchOption.AllDirectories);
+            var di = new DirectoryInfo(Utils.GaugeProjectRoot);
+            var allTopLevelDirs = di.GetDirectories("*", SearchOption.TopDirectoryOnly);
+            var projectDirs = new List<string> {"bin", "logs", "env", "gauge_bin", "obj", "reports"};
```

Comment:
we can set this in [`gauge_exclude_dirs`](https://docs.gauge.org/latest/configuration.html#default-properties-40) so that this can be user configured. Multivalued properties will be a comma separated value.